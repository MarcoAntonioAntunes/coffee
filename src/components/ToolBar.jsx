import React from 'react';

const ToolBar = ({ toggleButton }) => {
    return (
        <div className="location-toggle">
            <p>New York</p>
            <div className="toggle-button">
                <input onClick={toggleButton} className="tgl tgl-light" id="cb1" type="checkbox" />
                <label className="tgl-btn" htmlFor="cb1"></label>
            </div>
            <p>Dublin</p>
        </div>
    );
};

export default ToolBar;