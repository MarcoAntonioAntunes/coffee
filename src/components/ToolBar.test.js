import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ToolBar from './ToolBar';


it('should match the ToolBar  component to snapshot', () => {
    const component = renderer.create(
        <ToolBar />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

it('should render the ToolBar component', () => {
    configure({ adapter: new Adapter() });
    const component = shallow(
        <ToolBar />
    );
    const componentChildDiv = component.children().getElements()[1];
    expect(component.getElement().props.className).toEqual('location-toggle');
    expect(componentChildDiv.props.className).toEqual('toggle-button');
});