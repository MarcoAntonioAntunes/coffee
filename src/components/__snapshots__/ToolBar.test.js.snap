// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the ToolBar  component to snapshot 1`] = `
<div
  className="location-toggle"
>
  <p>
    New York
  </p>
  <div
    className="toggle-button"
  >
    <input
      className="tgl tgl-light"
      id="cb1"
      type="checkbox"
    />
    <label
      className="tgl-btn"
      htmlFor="cb1"
    />
  </div>
  <p>
    Dublin
  </p>
</div>
`;
